packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -393,10 +393,13 @@ export default class TradeStore extends BaseStore {
             }
         );
         reaction(
-            () => this.root_store.common.is_language_changing,
+            () => this.root_store.common.current_language,
```

Comment:
```suggestion
        when(
            () => [this.root_store.common.current_language && !this.amount],
```
best to use `when` if you have conditions like what hamza commented